src/lib.rs

```diff
@@ -165,9 +161,9 @@ impl Report {
 }
 
 /// A `TestSuite` groups together several [`TestCase`s](../struct.TestCase.html).
+#[derive(Debug, Clone, Getters)]
 pub struct TestSuite {
     name: String,
-    id: usize,
```

Comment:
These structs basically map to what is found inside the XML. If we remove this this is no longer true.

In the future I would like this libary to be able to read junit files via serde.

Reply:
Oh right, I forgot that this library is used in other direction too, my bad.